extern/alpaka/CMakeLists.txt

```diff
@@ -44,3 +44,11 @@ if( DEFINED _buildTestingValue )
 else()
    unset( BUILD_TESTING CACHE )
 endif()
+
+# Include the alpaka-functions.cmake file.
+# This has specific versions of the traccc-functions, with the add_library
+# and add_executable functions swapped to the alpaka versions.
+#
+# These versions ensure Alpaka flags are passed over, and the relevant
+# accelerator backends are enabled, including file properties.
+include( ${CMAKE_CURRENT_LIST_DIR}/cmake/alpaka-functions.cmake )
```

Comment:
Sounds reasonable!

I'm having a touch of trouble getting `device/alpaka/CMakeLists.txt` to pickup `traccc-alpaka-functions` though, which may just be that I'm missing something obvious. For now (to test the rest of the changes for your comments), I've updated the CMAKE Path in the main CMake file to include this path, as without it the path only includes the main CMAKE folder and the `_deps` folders in the build output.

Reply:
Very strange. 😕 This code is meant to take care of that:

https://github.com/acts-project/traccc/blob/main/CMakeLists.txt#L39-L41

That should make the `cmake/` directory visible to `include(...)` commands from there on out. Just as the `include(traccc-functions)` call demonstrates. 😕 

My best bet is a typo. 🤔 Since as long as

https://github.com/acts-project/traccc/blob/main/device/alpaka/CMakeLists.txt#L8

works correctly, then

```cmake
include( traccc-alpaka-functions )
```

on the next line should work as well... 🤔